controlplane/proto/controller/controller.proto

```diff
@@ -16,6 +16,7 @@ message ConfigRequest {
     repeated string bgp_peers = 2;
     string hash = 3;
     map<string, BgpPeers> bgp_peers_by_vrf = 4;
+    optional string agent_version = 5;
```

Comment:
Want to have it pass the the other build info too (version, commit sha, build date)?